react/src/lib/components/DeckGLMap/layers/map/mapLayer.stories.tsx

```diff
@@ -20,16 +20,18 @@ const valueRange = [-3071, 41048];
 const meshMapLayer = {
     "@@type": "MapLayer",
     id: "mesh-layer",
-    meshUrl: "hugin_depth_5_m.float32",
+    meshUrl:
+        "https://raw.githubusercontent.com/equinor/webviz-subsurface-components/master/react/src/demo/example-data/hugin_depth_5_m.float32",
```

Comment:
I don't think this change is needed

Reply:
This is a quite big map and I put it like this to make a more realistic loading time. Seems like it loads quite fast.